index.html

```diff
@@ -54,14 +60,13 @@
               <li class="list"><a class="a" href="#sponsor">Sponsor</a></li>
               <li class="list"><a class="a" href="">News</a></li>
               <li class="list listA">
-                <a class="list  cant" href="">CC Campaign</a>
+                <a class="list cant" href="">CC Campaign</a>
               </li>
             </ul>
           </div>
         </nav>
       </header>
 
-      <!-- Section 1 -->
       <section class="sectAA">
         <div class="tabq">"Hello! Sharing world"</div>
         <h1 class="het">Creative Commons Global Submit 2015</h1>
```

Comment:
- [ ] While working on the requested changes by the previous reviewer, However, your background image and background design are not showing as it is in [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

  ### Your Design
  ![Screenshot from 2023-03-17 10-21-06](https://user-images.githubusercontent.com/92303891/225881004-cbb28f84-84f1-44e3-9673-55638f6fc6ef.png)

   ### Cindys design
   
  ![image](https://user-images.githubusercontent.com/92303891/225881619-bc90360b-c448-474a-9c72-62b7eeb2b656.png)